From 43403f1d4744e0d71ef360655f44b489b57f443e Mon Sep 17 00:00:00 2001
From: Vasily Khoruzhick <anarsoul@gmail.com>
Date: Sun, 18 May 2014 12:43:35 +0300
Subject: [PATCH 02/17] ARM: s3c24xx: Remove S3C24XX_DMA dependency from
 S3C2410_DMA

S3C24XX_DMA enables legacy DMA driver, and since we want to
move to introduced generic dmaengine s3c24xx-dma driver,
it's necessary to remove this redundant dependency.

Signed-off-by: Vasily Khoruzhick <anarsoul@gmail.com>
---
 arch/arm/mach-s3c24xx/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-s3c24xx/Kconfig b/arch/arm/mach-s3c24xx/Kconfig
index d876431..3be853c 100644
--- a/arch/arm/mach-s3c24xx/Kconfig
+++ b/arch/arm/mach-s3c24xx/Kconfig
@@ -149,7 +149,6 @@ config S3C2410_DMA_DEBUG
 
 config S3C2410_DMA
 	bool
-	depends on S3C24XX_DMA && (CPU_S3C2410 || CPU_S3C2442)
 	help
 	  DMA device selection for S3C2410 and compatible CPUs
 
-- 
1.9.3

